zh_CN.ts

```diff
@@ -648,28 +648,28 @@ export const ZH = {
     PlayerTradeQuota: "剩余配额: %{value}",
     PlayerTradeUnavailable: "在与玩家进行交易之前，你需要在生产和存储中至少有一个资源",
     PlayerTradeToClaim: "您有 %{num} 宗玩家交易待领取",
-    PowerBank: "电力银行",
+    PowerBank: "电池组",
     PowerBankBuildDesc: "存储过剩的电力，在电力短缺时提供电能",
     PowerBankChargeSpeed: "充电速度",
     PowerBankPowerLeft: "电量剩余",
     PowerBankMoreCapacity: "电池保护",
-    PowerBankMoreCapacityDesc: "电力银行的充电速度只有原来50%，但是多50%容量",
-    PowerBankNotWorking: "⛔ 电力银行只在紧邻发电厂时工作",
+    PowerBankMoreCapacityDesc: "电池组的充电速度只有原来50%，但是多50%容量",
+    PowerBankNotWorking: "⛔ 电池组只在紧邻发电厂时工作",
     ExcludeBuildingsShutdown: "被关闭的建筑除外",
     ColorTheme: "游戏颜色主题",
     ColorThemeDesc: "颜色主题应用于游戏图标，高亮，网格和背景。需要重启生效",
     WarehouseBuildDesc: "以更低的燃料消耗运输及存储任何资源",
-    SiliconMine2xOutput: "Silicon Valley",
+    SiliconMine2xOutput: "硅谷",
     SiliconMine2xOutputDesc:
-        "Silicon mines produce 2x silicon but use 50% more power and silicon transport costs 50% more fuel",
-    CoalMine2xOutput: "Industrial Revolution",
-    CoalMine2xOutputDesc: "Coal mines produce 2x coal but use 50% more power and coal transport costs 50% more fuel",
-    AlMine2xOutput: "Aluminum Smelting",
-    AlMine2xOutputDesc: "Aluminum mines produce 2x aluminum and use 2x more power",
+        "硅矿场生产2倍的硅，但用电量增加50%，运输硅的燃料消耗增加50%",
+    CoalMine2xOutput: "工业革命",
+    CoalMine2xOutputDesc: "煤矿场生产2倍的煤，但用电量增加50%，运输煤的燃料消耗增加50%",
+    AlMine2xOutput: "铝电解",
+    AlMine2xOutputDesc: "铝矿场产生2倍的铝，但消耗2倍的电力",
     LoggingCamp2xOutput: "Deforestation",
```

Comment:
你忘了这个
You forgot this
```suggestion
    LoggingCamp2xOutput: "毁林",
```